.github/workflows/ruby.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [2.7, 3.0]
```

Comment:
I think we are already on Ruby 3.0 if we want to 🔥 the 2.7 build https://github.com/sul-dlss/puppet/blob/production/hieradata/node/mise-prod-a.stanford.edu.eyaml#L19